### PR TITLE
Fix typos :anchor:

### PR DESCRIPTION
**Pull request by rycus86**

### CODE REVIEW

File: .githooks/commit-msg/add-random-emoji [hunk 1]
```diff
@@ -13,6 +13,6 @@ fi
 # POSITION=$(($(od -A n -t d -N 1 /dev/urandom | tr -d ' ') % 11 + 1))
 SELECTED=":anchor:" #$(echo "$EMOJIS" | awk -v N=$POSITION '{ print $N }')
 
-sed -i -E '1s/(.*)\s*:\w+:)/\1/' "$1"
+sed -i -E '1s/(.*)\s*:\w+:/\1/' "$1"
```
Comment (rycus86) [inline .githooks/commit-msg/add-random-emoji:16]: Saw this error popping up in my terminal when executing this hook

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/nlohmann/json/master/LICENSE.MIT)
 [![GitHub Releases](https://img.shields.io/github/release/gabyx/githooks.svg)](https://github.com/gabyx/githooks/releases)
 
-**STILL BETA: Any changes with out notice!**
+**STILL BETA: Any changes without notice!**
```
Comment (rycus86) [inline README.md:11]: Spellchecker flagged this one up :)

**Comment by gabyx** [comment]
Great!. Will look it over!

File: .githooks/commit-msg/add-random-emoji (hunk 1 above)
Comment (gabyx) [reply .githooks/commit-msg/add-random-emoji:16]: Jeah its already fixe on another branch!, Thx :)!

File: README.md (hunk 2 above)
Comment (gabyx) [reply README.md:11]: Which one do you use? :-)

**Pull request merged**